homeassistant/components/remote/harmony.py

```diff
@@ -41,6 +41,8 @@
     vol.Required(CONF_NAME): cv.string,
     vol.Optional(ATTR_DELAY_SECS, default=DEFAULT_DELAY_SECS):
         vol.Coerce(float),
+    vol.Optional(ATTR_HOLD_SECS, default=DEFAULT_HOLD_SECS):
+        vol.Coerce(float),
```

Comment:
You extend the platform schema but I don't see that you ever read the configuration?

I am actually not sure that it makes sense to have a platform default for `hold_secs`, wouldn't it likely be different for each command?

Reply:
I don't understand why these are even part of the config, the config should just set up the connection and that's it ? 

Let's not add this one, in a future PR remove the other ones.

Reply:
Makes sense, removed it. I'll open a future PR to remove the other one (after I'm done with XMPP for aioharmony, just wanted to take a quick break from that one).